heroku/cli
# refactor: migrate container:release to oclif/core

## Testing
- Check out this branch and run `yarn` and `yarn build`
- Either use a Docker test app you've already created or create a new one. Make sure it has been pushed to Heroku.
- run `./bin/run container:release web -a APP-NAME`
